.aios-core/infrastructure/scripts/llm-routing/templates/claude-free.sh

```
@@ -0,0 +1,57 @@
+        dir="$(dirname "$dir")"
+    done
+    return 1
+}
+
+# Try to load from .env
+PROJECT_ROOT=$(find_env)
+
+if [[ -n "$PROJECT_ROOT" ]]; then
+    # Load DEEPSEEK_API_KEY from .env
+    if [[ -f "$PROJECT_ROOT/.env" ]]; then
+        export $(grep -E '^DEEPSEEK_API_KEY=' "$PROJECT_ROOT/.env" | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)